pyatlan/client/atlan.py

```diff
@@ -1,14 +1,16 @@
 # SPDX-License-Identifier: Apache-2.0
 # Copyright 2022 Atlan Pte. Ltd.
 # Based on original code from https://github.com/apache/atlas (under Apache-2.0 license)
+from __future__ import annotations
+
 import abc
 import contextlib
 import copy
 import json
 import logging
 import os
 from abc import ABC
-from typing import ClassVar, Generator, Optional, Type, TypeVar, Union
+from typing import ClassVar, Generator, List, Optional, Type, TypeVar, Union
```

Comment:
You might be right, this could be left-over from using `ForwardRef()` to try to workaround `validate_arguments()` lack of support for forward references (which didn't end up working either, anyway, and seems to be an anti-pattern in Python despite being suggested in Pydantic's own docs)

Ref:
- Python docs (note from PEP 585 you must use `List` and not `list`): https://docs.python.org/3.11/library/typing.html#typing.ForwardRef
- Pydantic's docs: https://docs.pydantic.dev/1.10/usage/postponed_annotations/

Reply:
Yup, looks like it's unnecessary with the current approach — have reverted in latest commit I pushed